# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1455124896 18000
#      Wed Feb 10 12:21:36 2016 -0500
# Node ID bae38398f5f1fed7198a9494eede4fcde1082c04
# Parent  7af5ca01ecaccbe19a8103a06bef62f8efddebee
use consistent style for all liboctave and libinterp include guards

* mex.h, mexproto.h, mxarray.in.h, oct-obj.h, octave-default-image.h:
Use include guard style that is consistent with other files in Octave.

diff --git a/libinterp/corefcn/mex.h b/libinterp/corefcn/mex.h
--- a/libinterp/corefcn/mex.h
+++ b/libinterp/corefcn/mex.h
@@ -40,18 +40,18 @@ HOWEVER CAUSED AND ON ANY THEORY OF LIAB
 LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
 OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
 SUCH DAMAGE.
 
 */
 
 /* mex.h is for use in C-programs only; do NOT include it in mex.cc */
 
-#if ! defined (MEX_H)
-#define MEX_H
+#if ! defined (octave_mex_h)
+#define octave_mex_h 1
 
 #define HAVE_OCTAVE
 
 typedef void mxArray;
 
 #if ! defined (__cplusplus) && ! defined (bool)
 typedef int bool;
 #endif
diff --git a/libinterp/corefcn/mexproto.h b/libinterp/corefcn/mexproto.h
--- a/libinterp/corefcn/mexproto.h
+++ b/libinterp/corefcn/mexproto.h
@@ -40,18 +40,18 @@ HOWEVER CAUSED AND ON ANY THEORY OF LIAB
 LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
 OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
 SUCH DAMAGE.
 
 */
 
 /* mex.h is for use in C-programs only; do NOT include it in mex.cc */
 
-#if ! defined (MEXPROTO_H)
-#define MEXPROTO_H
+#if ! defined (octave_mexproto_h)
+#define octave_mexproto_h 1
 
 #if defined (__cplusplus)
 #  include <cstdlib>
 extern "C" {
 #else
 #  include <stdlib.h>
 #endif
 
diff --git a/libinterp/corefcn/mxarray.in.h b/libinterp/corefcn/mxarray.in.h
--- a/libinterp/corefcn/mxarray.in.h
+++ b/libinterp/corefcn/mxarray.in.h
@@ -39,18 +39,18 @@ DAMAGES (INCLUDING, BUT NOT LIMITED TO, 
 OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
 HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
 LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
 OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
 SUCH DAMAGE.
 
 */
 
-#if ! defined (MXARRAY_H)
-#define MXARRAY_H
+#if ! defined (octave_mxarray_h)
+#define octave_mxarray_h 1
 
 typedef enum
 {
   mxREAL = 0,
   mxCOMPLEX = 1
 }
 mxComplexity;
 
diff --git a/libinterp/corefcn/oct-obj.h b/libinterp/corefcn/oct-obj.h
--- a/libinterp/corefcn/oct-obj.h
+++ b/libinterp/corefcn/oct-obj.h
@@ -20,11 +20,16 @@ along with Octave; see the file COPYING.
 
 */
 
 ////////////////////////////////////////////////////////////////////////
 // Note: oct-obj.h was deprecated in version 4.2 and will be removed in
 // version 4.6.
 ////////////////////////////////////////////////////////////////////////
 
+#if ! defined (octave_oct_obj_h)
+#define octave_oct_obj_h 1
+
 #warning oct-obj.h has been deprecated; use ovl.h instead
 
 #include "ovl.h"
+
+#endif
diff --git a/libinterp/corefcn/octave-default-image.h b/libinterp/corefcn/octave-default-image.h
--- a/libinterp/corefcn/octave-default-image.h
+++ b/libinterp/corefcn/octave-default-image.h
@@ -15,16 +15,19 @@ FITNESS FOR A PARTICULAR PURPOSE.  See t
 for more details.
 
 You should have received a copy of the GNU General Public License
 along with Octave; see the file COPYING.  If not, see
 <http://www.gnu.org/licenses/>.
 
 */
 
+#if ! defined (octave_octave_default_image_h)
+#define octave_octave_default_image_h 1
+
 static char default_im_data[] =
 {
   31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
   31,31,31,31,31,31,20,20,20,20,20,20,21,21,21,21,
   31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
   31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
   31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
   31,31,31,20,20,20,20,20,20,20,20,20,20,20,21,21,
@@ -275,8 +278,9 @@ static char default_im_data[] =
   31,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,
   19,19,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
   31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
   31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
   31,31,31,31,31,19,19,19,19,19,19,19,19,19,19,31,
   31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31
 };
 
+#endif
